logic/common/tiled_utils.py

```diff
@@ -4,72 +4,70 @@
 import zlib
 import numpy
 
+
+
+#--------------------------------------------------#
+'''Base64 string <-> Array'''
+
 def DecodeIntoTiles2d(encoded_str, row_width):
-    '''Takes a TILED layer encoded string and converts it into a tiles2d
-    
-    :param encoded_str: a TILED layer's 'data'. This is an encoded string using zlib base64 compression
-    :param row_width: integer indicating how many cells each row should have in the resultant tiles2d
-    :returns tiles2d: A 2d array of tile IDs
-    
-    Example: 
-    >>> DecodeIntoTiles2d("eAENw4cNACAMAKA66/r/XiGhRES12R1O0+X2eH1+BeAATw==", 4)
-    >>> [[1, 2, 3, 4], [5, 6, 7, 8], [9, 10, 11, 12]]
     '''
-    
-    # Decode the base64 string
+    Takes a TILED layer encoded string and converts it into a tiles2d
+      encoded_str - string containing a layer's 'data' in TILED, zlib-encoded and base64-compressed
+      row_width   - integer indicating number of cells each row should contain
+      tiles2d     - 2d array of Tile IDs, first row is at top
+      >>> DecodeIntoTiles2d("eAENw4cNACAMAKA66/r/XiGhRES12R1O0+X2eH1+BeAATw==", 4)
+      >>> [[1, 2, 3, 4], [5, 6, 7, 8], [9, 10, 11, 12]]
+    '''
+
+    # Convert the base64 string to a numpy array (uint32)
     decoded_data = base64.b64decode(encoded_str)
-    
-    # Decompress the data using zlib
     decompressed_data = zlib.decompress(decoded_data)
-    
-    # Convert the decompressed data to a numpy array with type uint32
     array = numpy.frombuffer(decompressed_data, dtype=numpy.uint32)
-    
+
     # Reshape the array to the desired dimensions
     tiles2d = array.reshape(-1, row_width)
-    
     return tiles2d.tolist()
-    
+
+
+
 def EncodeIntoZlibString64(tiles2d):
-    ''' Takes a tiles2d and converts it into a zlib base64 encoded string.
-    
-    :param tiles2d: A 2d array of tile IDs
-    :returns encoded_string: a zlib base64 string
-        
-    Example: 
-    >>> EncodeIntoZlibString64([[1, 2, 3, 4], [5, 6, 7, 8], [9, 10, 11, 12]])
-    >>> "eAENw4cNACAMAKA66/r/XiGhRES12R1O0+X2eH1+BeAATw=="
     '''
-    # Convert the input array to a numpy array with type uint32
-    np_array = numpy.array(tiles2d, dtype=numpy.uint32)
+    Takes a TILED layer encoded string and converts it into a tiles2d
+      tiles2d     - 2d array of Tile IDs, first row is at top
```

Comment:
I'm not a fan of this type of spacing - it creates unseemly gaps in the description IMO. Let's just use one space after the variable name